hosting/static/hosting/js/initial.js

```diff
@@ -153,4 +153,83 @@ $( document ).ready(function() {
             $('.navbar-fixed-top.topnav').css('padding-right', topnavPadding-scrollbarWidth);
         }
     });
+
+    /* ---------------------------------------------
+     Scripts initialization
+     --------------------------------------------- */
+    var cardPricing = {
+        'cpu': {
+            'id': 'coreValue',
+            'value': 1,
+            'min': 1,
+            'max': 48,
+            'interval': 1
```

Comment:
We can add this here as is, but we should create another task to query the limits from the backend or to use a javascript template so that the actual min/max values are retrieved from the database. 

Can you add a task for it, please?

Reply:
Sure! Added one here
https://redmine.ungleich.ch/issues/4545